src/audio/base_fw.c

```diff
@@ -391,6 +394,42 @@ static int basefw_power_state_info_get(uint32_t *data_offset, char *data)
 	return 0;
 }
 
+static int fw_config_set_force_l1_exit(const struct sof_tlv *tlv)
+{
+	const uint32_t force = tlv->value[0];
+
+	if (force) {
+		tr_info(&basefw_comp_tr, "FW config set force dmi l0 state");
+#if defined(CONFIG_SOC_SERIES_INTEL_ACE)
```

Comment:
Not sure what to return if the functionality is not available. Alternatively you could #ifdef the whole funciton body and at the end:

```
#if defined(CONFIG_SOC_SERIES_INTEL_ACE)
const uint32_t force = ..
...
#else
return IPC4_UNAVAILABLE;
#endif
```